package.json

```diff
@@ -4,11 +4,14 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "serve": "vite preview"
+    "serve": "vite preview",
+    "postinstall": "patch-package"
```

Comment:
That's cool. I've not seen patches applied this way before.

Reply:
I saw it mentioned on twitter a couple of months ago and I was just waiting for the opportunity to use it 😄